docker/iml-timer.dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM rust-iml-base as builder
+
+FROM imlteam/systemd-base
+COPY --from=builder /build/target/release/iml-timer /bin/
+COPY docker/iml-timer/iml-timer.service /etc/systemd/system/
+COPY docker/iml-timer/start-stratagem-scan.py /usr/local/bin
+
+
+RUN yum install -y epel-release \
+  && yum install -y python2 \
+  && pip2 install requests \
+  && systemctl enable iml-timer
```

Comment:
There is no cleanup here for pip, the cache adds around 20MB. N.B. this doesn't matter if the command was rust, as we'd have no external deps.

Reply:
right, i'll get rid of the python file and can remove pip entirely.